src/utils/error.const.ts

```diff
@@ -0,0 +1,5 @@
+// https://ecmps.camdsupport.com/documents/ECMPS%20Import%20Check%20Specifications%202018Q2.pdf
+export const IMPORT_CHECK_ERROR = {
+  IMPORT_23:
+    'You have reported a date in a Daily Summary, DailyTest Summary or Hourly Operating record that does not fall within the reporting period. The emissions file will not be imported.',
```

Comment:
I feel like we need a consistent approach to this. The above will work for this check but there are other checks which have multiple results like we talked about earlier (Result A, Result B, ...etc). I was also under the assumption that we'd we'd have to call a function which would return us the string so in this case it, when this import_23 is used it'd look something like, `return [IMPORT_CHECK_ERROR.IMPORT_23.resultA()]

Even though in this case we can just do  IMPORT_CHECK_ERROR.IMPORT_23, I think we should be consistent with what future checks would do.

Reply:
Sure, I wasn't too concerned with consistency because it's just a constant accessor. I'll throw a RESULT_A() in there.